pymilo/pymilo_param.py

```diff
@@ -57,14 +57,20 @@
 from sklearn.cluster import FeatureAgglomeration
 from sklearn.cluster import DBSCAN
 
+bisecting_kmeans_support = False
+try:
+    from sklearn.cluster import BisectingKMeans
+    bisecting_kmeans_support = True
+except BaseException:
+    print("BisectingKmeans doesn't exist in this version of python.")
```

Comment:
The information provided in this message is not precise enough.
`BisectingKmeans doesn't exist in this version of scikit-learn`

Reply:
we decided to remove this log.